src/main/java/frc/robot/subsystems/LiftSystem.java

```diff
@@ -0,0 +1,147 @@
+
+    motorTwo = new CANSparkMax(MOTOR_RIGHT, MotorType.kBrushless);
+    motorTwo.setInverted(true);
+
+    armEncoder = new DutyCycleEncoder(0);
+  
+    liftGroup = new MotorControllerGroup(motorOne, motorTwo);
+
+    //Setting default values for PID
+    pControllerOne = new PIDController(0.25, 0.0, 0.0);
+    pControllerTwo = new PIDController(0.25, 0.0, 0.0);
+
+    //Limit Switches
+    limitUp = new DigitalInput(LIMIT_SWITCH_UP);
+    limitDown = new DigitalInput(LIMIT_SWITCH_DOWN);
+
+  }
+
+  /**
+   * @param xboxController Operator
+   * @return Command that uses obtained values from the limit switch to lift the arm within a specified range.
+   */
+  public CommandBase liftArms(XboxController xboxController){
+
+    return runEnd(
+      () -> {
+        if(limitUp.get() && (xboxController.getLeftY() > 0)) { // When the upward limit switch is triggered and the operator attempts to move upward, it will not move upward.
+          liftGroup.set(0);
+        } else if(limitDown.get() && (xboxController.getLeftY() < 0)) { // When the downward limit switch is triggered and the operator attempts to move downward, it will not move downward.
+          liftGroup.set(0);
+        } else {
+          liftGroup.set(-0.2 * xboxController.getLeftY());
+        }
+      },
+
+      () -> {
+        liftGroup.set(0);
+      }
+    );
+  }
+
+  /**
+   * @param position to go to
+   * @return Command that uses PID to lift the gripper to the specified position
+   */
+  public CommandBase liftArmsToPosition(double position){
+
+    double clampedPos = MathUtil.clamp(position, MIN_POSITION, MAX_POSITION);
+    SmartDashboard.putNumber("setpoint", clampedPos);
+
+    return runEnd(
+    //Runs repeatedly until the end
+    () -> {
+      
+      motorOne.set(pControllerOne.calculate(getPosition(), clampedPos));
+      motorTwo.set(pControllerTwo.calculate(getPosition(), clampedPos));
+
+      System.out.println("Inside lift command reached");
+    }, 
+    //Runs when command ends
+    () -> {
+      //liftArms(0);
+    }
+    ).until(
+      () -> {
+        return (getPosition() > clampedPos && getPosition() < clampedPos);
+      }
+    );
+  }
+
+
+  public double getPosition(){
+    return armEncoder.getAbsolutePosition();
+  }
+
+  @Override
+  public void initSendable(SendableBuilder builder) {
+    builder.addDoubleProperty("Encoder", this::getPosition, null);
+    builder.addDoubleProperty("motor velocity", motorOne.getEncoder()::getVelocity, null);
+
+    builder.addBooleanProperty("Up Limit Switch", () -> !limitDown.get(), null);
+    builder.addBooleanProperty("Down Limit Switch", () -> !limitUp.get(), null);
+  }
+
+  @Override
+  public void periodic() {
+    // This method will be called once per scheduler run
+  }
+
+  @Override
+  public List<Connection> hardwareConnections(){
+    return List.of(
+      Connection.fromSparkMax(motorOne),
+      Connection.fromSparkMax(motorTwo)
+    );
+  }
```

Comment:
discuss how to implement this for limit switches, encoders, etc
additionally implement autonomous function test